build.gradle.kts

```diff
@@ -66,7 +69,7 @@ version = run {
             }
         } else {
             // do not care
-            ""
+            "-SNAPSHOT"
```

Comment:
I'd personally leave it empty, this code is not reachable I guess, as in teamcity we have only two types of builds: release and nighlty